runtime/onert/sample/minimal-python/src/minimal.py

```diff
@@ -0,0 +1,65 @@
+from nnfwapi.libnnfw_api_pybind import *
+import numpy as np
+import sys
+
+
+def num_elems(tensor_info):
+    """Get the total number of elements in nnfw_tensorinfo.dims."""
+    n = 1
+    for x in range(tensor_info.rank):
+        n *= tensor_info.dims[x]
+    return n
+
+
+def main(NNPACKAGE_PATH, BACKEND="cpu", OPERATION=""):
+    # Create session and load nnpackage
+    # OPERATION is optional for assigning a specific backend to operations.
+    # "cpu" is default value of BACKEND.
+    if OPERATION:
+        session = nnfw_session(NNPACKAGE_PATH, BACKEND, OPERATION)
+    else:
+        session = nnfw_session(NNPACKAGE_PATH, BACKEND)
```

Comment:
```suggestion
def main(NNPACKAGE_PATH, BACKENDS="cpu", OPERATIONS=""):
    # Create session and load nnpackage
    # OPERATIONS is optional to assign a specific backend to each operation.
    # BACKENDS's default value is "cpu".
    if OPERATIONS:
        session = nnfw_session(NNPACKAGE_PATH, BACKENDS, OPERATIONS)
    else:
        session = nnfw_session(NNPACKAGE_PATH, BACKENDS)
```

Reply:
Writing variables in all uppercase is different from the universal coding convention I know. Is there a reason why you wrote it this way?

Reply:
For no specific reason, we simply used it for emphasis.
We'll change the variables to lowercase by reflecting your feedback.